rust/psibase/src/hex.rs

```diff
@@ -4,12 +4,14 @@ use std::marker::PhantomData;
 use std::ops::Deref;
 use std::str::FromStr;
 
-trait ToHex:
+pub trait ToHex:
```

Comment:
This should remain a private implementation detail. The ToString (implicit from Display) and FromStr traits provide conversion.

Reply:
I did this to enable `Vec<u8>::to_hex()` for debugging purposes. But I will rollback...